.dvcignore

```diff
@@ -0,0 +1,2 @@
+# add patterns of files dvc should ignore, can improve performance
```

Comment:
does DVC creates this file automatically? Probably we don't need it in this PR?

Reply:
I suppose I accidentally run ``dvc init`` and this file was generated. I'll delete it.